crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -277,6 +277,9 @@ impl<'w> EntityMut<'w> {
     }
 
     // TODO: move to BundleInfo
+    /// Removes a [`Bundle`] of components from the entity and returns the bundle.
```

Comment:
I know this is technically correct, now that components are bundles, but it feels a bit less obvious that this can also be used to remove components

Reply:
I considered specifying that it could be used with components, but none of the docs on `EntityCommands`' methods mentioned components other than the deprecated ones.